dips/dip-1.mdx

```diff
@@ -211,18 +211,88 @@ Represents an error that occurred in response to a Command.
 {
     "type": "command_error",
     "field": "payment.sender.kyc_data.surname",
-    "code": "missing_data",
+    "code": "missing_field",
     "message": "",
 }
 ```
-`command_error` occurs in response to a Command failing to be applied - for example, a high level validation error.  `protocol_error` occurs in response to a failure related to the lower-level protocol.  Note: Additional information detailing all errors will be added.
 
+`command_error` occurs in response to a Command failing to be applied - for example, a high level validation error.  `protocol_error` occurs in response to a failure related to the lower-level protocol.
```

Comment:
let's be more explicit with these errors: 
command_error results committed failed commands, but commands with protocol_error are retry-able.

Reply:
can we specify their types of every error below?

Reply:
I'm not up to mark anything "retry-able". 
All errors are all retry-able, for example, for `missing_field`: it is possible server side introduced a bug to raise the error, once error rates increased, the deployment of bug code is reverted, then client side retry can resolve the error and move things forward.